CHANGELOG

```diff
@@ -0,0 +1,10 @@
+Arduino_MKRENV ?.?.? - ????.??.??
+
+Arduino_MKRENV 1.1.0 - 2019.05.31
+
+* Added imperial units support
+* Added readIlluminace() API to replace readLux()
```

Comment:
There is a small typo here: `readIlluminace()` -> `readIlluminance()`

Reply:
done